lib/pbench/cli/agent/__init__.py

```diff
@@ -0,0 +1,20 @@
+"""
+Create a click context object that holds the state of the agent
+invocation. The CliContext will keep track of passed parameters,
+what command created it, which resources need to be cleaned up,
+and etc.
+We create an empty object at the begining and populate the object
```

Comment:
An empty line between paragraphs would make it easier to read.

Reply:
Done